openedx/features/enterprise_support/api.py

```diff
@@ -477,14 +478,20 @@ def is_course_in_enterprise_catalog(site, course_id, user, enterprise_catalog_id
     )
     response = cache.get(cache_key)
     if not response:
-        try:
-            # GET: /api/v1/catalogs/{catalog_id}/contains?course_run_id={course_run_ids}
-            response = course_discovery_api_client(user=user).catalogs(enterprise_catalog_id).contains.get(
-                course_run_id=course_id
-            )
-            cache.set(cache_key, response, settings.COURSES_API_CACHE_TIMEOUT)
-        except (ConnectionError, SlumberBaseException, Timeout):
-            LOGGER.exception('Unable to connect to Course Catalog service for catalog contains endpoint.')
+        catalog_integration = CatalogIntegration.current()
+        if catalog_integration.enabled:
```

Comment:
Do we need to log in case we are using enterprise but `catalog_integration` is not enabled?